sageopt/symbolic/polynomials.py

```diff
@@ -236,7 +236,7 @@ def __mul__(self, other):
 
     def __truediv__(self, other):
         if not isinstance(other, __NUMERIC_TYPES__):  # pragma: no cover
-            raise RuntimeError('Cannot divide a polynomial by the non-numeric type: ' + type(other) + '.')
+            raise RuntimeError('Cannot divide a polynomial by the non-numeric type: ' + str(type(other)) + '.')
```

Comment:
Did you write a unittest that hits this line? If so, you can remove the ``  # pragma: no-cover`` comment on the previous line.

Reply:
Yes, I'll remove the `# pragma: no-cover` comment on that line and also on line 225 in the`__mul__` function.